docs/partial_source/contributing/1_the_basics.rst

```diff
@@ -135,6 +135,29 @@ investigate!
 For questions, please reach out on the `contributing discussion`_
 or on `discord`_ in the `todo list issues channel`_!
 
+Extensions
+----------
+
+We primarily invite contributors to contribute to the tasks listed as :ref:`Open Tasks` as they are
+on our roadmap currently. As a result of this we mainly recommend contributions which fall under the
+:ref:`Open Tasks`. However, if a contributor would like to extend Ivy's functionality with new 
+functions, submodules, etc. following should ideally be the workflow:
```

Comment:
then the workflow should ideally be the following:

Reply:
Done